test/abstract_syntax_tree.coffee

```diff
@@ -164,21 +164,36 @@ test "AST as expected for NaNLiteral node", ->
     type: 'Identifier'
     name: 'NaN'
 
-# test "AST as expected for StringLiteral node", ->
-#   testExpression '"string cheese"',
-#     type: 'StringLiteral'
-#     value: '"string cheese"'
-#     quote: '"'
-
-#   testExpression "'cheese string'",
-#     type: 'StringLiteral'
-#     value: "'cheese string'"
-#     quote: "'"
-
-# test "AST as expected for RegexLiteral node", ->
-#   testExpression '/^(?!.*(.).*\\1)[gimsuy]*$/',
-#     type: 'RegexLiteral'
-#     value: '/^(?!.*(.).*\\1)[gimsuy]*$/'
+test "AST as expected for StringLiteral node", ->
+  # Just a standalone string literal would be treated as a directive
```

Comment:
“would be treated as directive”? You mean as `StringLiteral`?

Reply:
Sorry, that comment probably needed more explanation

Expanded to "Just a standalone string literal would be treated as a directive, so embed the string literal in an enclosing expression (e.g. a call)."

Meaning that if this checked the AST for just eg `"string cheese"`, it wouldn't be able to test for a `StringLiteral` AST node because that single string would actually be "parsed" as a directive (so would generate a `Directive` AST node)